docs/quick-start.md

```diff
@@ -145,4 +122,4 @@ function chatQuery(query) {
 
 ### Known issues
 
-Asynchronous support in Django is fairly new, as a result of this WagtailVectorIndexSSEConsumer can't tell when a client disconnects from an event-stream, This may result in queries being processed by the server as zombie threads. Support for handling [client disconnects](https://docs.djangoproject.com/en/dev/topics/async/#handling-disconnects) was added in Django 5.0, although currently untested in this package.
+Asynchronous support in Django is fairly new, as a result of this WagtailVectorIndexSSEConsumer can't tell when a client disconnects from an event-stream, This may result in queries being processed by the server as zombie threads.
```

Comment:
```suggestion
Asynchronous support in Django is fairly new and `WagtailVectorIndexSSEConsumer` can't tell when a client disconnects from an event-stream. This may result in queries being processed by the server as zombie threads.
```